CodeEdit/Features/Documents/Controllers/CodeEditSplitViewController.swift

```diff
@@ -46,8 +46,8 @@ final class CodeEditSplitViewController: NSSplitViewController {
 
     override func viewWillAppear() {
         super.viewWillAppear()
-        let width = workspace.getFromWorkspaceState(key: self.widthStateName) as? CGFloat ?? 0
-        splitView.setPosition(width, ofDividerAt: .zero)
+        let width = workspace.getFromWorkspaceState(key: self.widthStateName) as? CGFloat
+        splitView.setPosition(width ?? CodeEditWindowController.minSidebarWidth, ofDividerAt: .zero)
```

Comment:
Shouldn't the default width for the sidebar be `snapWidth` instead of the minimum width? I believe this is also what we are doing for now.

Reply:
Yes, if no saved width exists, it should default to `snapWidth`